src/main/resources/anniv5Resources/summaries/ratings.txt

```diff
@@ -11,6 +11,7 @@ BellordPack	2	3	2	4	1
 BitingColdPack	3	2	2	3	2
 BlacksmithsPack	3	3	4	4	4
 BoardGamePack	4	3	2	4	2
+BulwarkPack	2	5	1	4	4
```

Comment:
The first 5 in Defense! Also a 1 in Support, because it doesn't even have anything that draws cards or makes energy, which feels appropriate for a very focused pack like this one.